arrow-optics/src/main/kotlin/arrow/optics/std/set.kt

```diff
@@ -4,10 +4,12 @@ import arrow.core.SetK
 import arrow.core.SetExtensions
 import arrow.core.identity
 import arrow.core.k
+import arrow.optics.typeclasses.At
 
 /**
  * [PIso] that defines the equality between a [Set] and a [SetK]
  */
+@Deprecated
```

Comment:
☝️ 

Pretty sure we will be getting rid of these as we do not want to project anything over SetExtensions. However, if SetK is being deprecated not sure if we want to generate an alternative over Set

Reply:
SetK is being deprecated, there is no replacement